app/controllers/course/leaderboards_controller.rb

```diff
@@ -3,6 +3,7 @@ class Course::LeaderboardsController < Course::ComponentController
   before_action :check_component
   before_action :load_settings
   before_action :add_leaderboard_breadcrumb
+  helper Course::Achievement::ControllerHelper.name.sub(/Helper$/, '')
```

Comment:
I think there are a few controllers now that use the Achievement Controller Helper. It might make more sense for those controllers to just use their own controller, and the achievement controller helper's common functionality extracted into a separate module that the achievement controller helper itself includes.